place_and_route/private/export_def.bzl

```diff
@@ -0,0 +1,58 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Export to def openROAD commands"""
+
+load("//place_and_route/open_road.bzl", "OpenRoadInfo", "merge_open_road_info", "openroad_command")
```

Comment:
@mithro This is incorrect.

```
load("//place_and_route/open_road.bzl", "OpenRoadInfo", "merge_open_road_info", "openroad_command")
```

It should be 
```
load("//place_and_route:open_road.bzl", "OpenRoadInfo", "merge_open_road_info", "openroad_command")

```

Reply:
Fixed.